pkg/controller/migmigration/migrate.go

```diff
@@ -86,7 +87,16 @@ func (r *ReconcileMigMigration) migrate(migration *migapi.MigMigration) (time.Du
 	}
 
 	// Result
-	migration.Status.Phase = task.Phase
+	if migration.Status.Phase != task.Phase {
+		migration.Status.Phase = task.Phase
+		if migration.Status.Phase != Created {
+			r.recorder.Eventf(
+				migration,
+				v1.EventTypeNormal,
+				migration.Status.Phase,
+				"Step completed.")
```

Comment:
I wonder whether it might be better to use a message more specific to the step in question (i.e. "Step 2 of 22 completed") You could use task.getStep() to pull the number information out, although you'd have to be careful to get the numbers for the last-completed step rather than the step that's just beginning.